crates/hyperdrive-math/src/long/max.rs

```diff
@@ -340,7 +340,7 @@ impl State {
     /// It's possible that the pool is insolvent after opening a long. In this
     /// case, we return `None` since the fixed point library can't represent
     /// negative numbers.
-    fn solvency_after_long(
+    pub fn solvency_after_long(
```

Comment:
This comment goes for the other functions in this file. My guess is that you want to use these in the `targeted` long module, but these shouldn't be accessible to someone that is using the SDK because it will clutter up the modules that they will see. A way to accomplish both goals is to add a modifier to `pub`. In this case, I think that something like `pub(super)` is appropriate. This allows the function to be imported within the `long` module, but it is considered private outside of that module.